packages/clay-tabs/src/index.tsx

```diff
@@ -10,38 +10,35 @@ import Content from './Content';
 import Item from './Item';
 import TabPane from './TabPane';
 
+export type DisplayType = null | 'tabs' | 'underline';
```

Comment:
I think in this case we can go with different names for the display types, maybe just change `tabs` to `basic`, I didn't have much creativity, if you have another name feel free.